src/rp2_common/pico_audio_i2s/audio_i2s.pio

```diff
@@ -54,8 +73,8 @@ static inline void audio_i2s_program_init(PIO pio, uint sm, uint offset, uint da
 
     pio_sm_init(pio, sm, offset, &sm_config);
 
-    uint pin_mask = (1u << data_pin) | (3u << clock_pin_base);
-    pio_sm_set_pindirs_with_mask(pio, sm, pin_mask, pin_mask);
+    uint64_t pin_mask = (1ull << data_pin) | (3ull << clock_pin_base);
```

Comment:
again, whilst this code is perfectly fine, for the sake of the many massively code constrained RP2040 things that use `pico_i2s_audio` I think i'd keep the old code in an `#else` of a `#if NUM_GPIOS > 32`

Reply:
@jepler Just to slightly clarify Graham's comment here: the use of `uint64_t` will force the inclusion of 64-bit math code, which will make the resultant firmware binary bigger (for everybody). Whereas it'd be better to stick to `uint32_t` (or similar) where possible, and only have 64-bit math be used when turned on by a preprocessor `#define`.

Reply:
I now avoid the call to pio_set_gpio_base and the use of uint64_t & pio_sm_set_pindirs_with_mask64 when PICO_PIO_USE_GPIO_BASE is not set.